Tasks/GulpV0/task.json

```diff
@@ -15,8 +15,8 @@
     "author": "Microsoft Corporation",
     "version": {
```

Comment:
Not sure if you saw my comment on the thread in this PR (now resolved), so posing here just in case.

What kind of versioning scheme is used? If it's semantic versioning, the major would need to be bumped for a breaking change, right? https://semver.org/ Ignore me if this isn't employed though. Just wanted to double check.

Reply:
I believe it is using semver, but as we're on major-version 0, "anything goes":

> 4. Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.

Reply:
That said, this does risk breaking some unknown number of users in order to fix a different oversight.

Reply:
If it were me, I'd probably bump the major then to avoid any potential customer dissatisfaction/confusion. "Why did this stop working all of a sudden", type questions.

I don't think there are any cons to bumping the major version, right? People can't use the local Gulp versions anyway, so if they want this capability, then upgrading to the newer version should be OK.

Reply:
Had a talk with someone offline, and they were in agreement that there's no reason the Gulp task should still be v0 anyway. I'll take care of it.